sympy/vector/parametricregion.py

```diff
@@ -0,0 +1,48 @@
+from sympy.core.basic import Basic
+
+class ParametricRegion(Basic):
+    """
+    Represents a parametric region in space.
+    """
+    def __init__(self, system, *args):
+        """
+        Examples
+        ========
+
+        >>> from sympy import symbols, cos, sin, pi
+        >>> from sympy.vector import CoordSys3D, ParametricRegion
+        >>> r, theta = symbols("r theta")
+        >>> C = CoordSys3D('C')
+        >>> circle = ParametricRegion(C, r*cos(theta), r*sin(theta), (theta, 0, 2*pi))
+        """
+        self._definition = list()
+        self._bounds = list()
+        self._parameters = list()
+        self.system = system
```

Comment:
`self._system = system` would be the standard way... then
 ```
@property 
def system(self):
    return self._system
```

This make sure that `system` cannot be modified by the user.

Reply:
Yes, I will correct it.